docs/08-developer-guide/01-build/01-maven.md

```diff
@@ -0,0 +1,129 @@
+    <br/>Obtain the Ozone sources from the [download](/download) page.
+    <br/>Next, unpack the tarball
+    ```bash
+    tar xzf ozone-<version>-src.tar.gz
+    cd ozone-<version>-src
+    ```
+  </TabItem>
+</Tabs>
+
+## Build the Project
+
+Apache Ozone uses Maven as its build system. The build process compiles the source code, runs tests, and creates deployable artifacts. The project supports various build configurations to accommodate different development and deployment needs.
+
+### Build Options
+
+The build system offers several options to customize the build process according to your requirements:
+
+#### Basic Build
+
+For a basic build that skips tests:
+
+```bash
+mvn clean install -DskipTests=true
+```
+
+This command will:
+
+- Clean previous build artifacts
+- Compile the source code
+- Package the compiled code into JAR files
+- Create a distribution in `hadoop-ozone/dist/target/ozone-<version>`
+
+#### Build with Tests
+
+To run unit tests during the build:
+
+```bash
+mvn clean install
+```
+
+#### Create Distribution Tarball
+
+To create a distribution tarball for deployment:
+
+```bash
+mvn clean install -DskipTests=true -Pdist
+```
+
+This creates a tarball in `hadoop-ozone/dist/target` that contains all necessary files for deployment.
+
+#### Maven Build Options
+
+Several Maven options are available to customize the build process:
+
+Here's the conversion to a markdown table:
+
+| Command | Description |
+|---------|-------------|
+| `-DskipTests=true` | Skip all tests |
+| `-Dskip.installnpm -Dskip.installnpx -Dskip.installyarn -Dskip.npm -Dskip.npx -Dskip.yarn` | Skip building the Javascript frontend for Recon |
+| `-Pdist` | Enable the distribution profile to create deployment tarballs |
+| `-T 4` | Use 4 threads for parallel building (adjust number based on your CPU) |
+| `-T 2C` | Use 2 threads per core for parallel building (adjust number based on your CPU) |
+| `-am -pl module-name` | Build a specific module and its dependencies |
+| `-DskipShade` | Skip shading. This saves a ton of time by e.g. skipping the ozone-filesystem-hadoop3 fat jar build |
+| `-Dmaven.artifact.threads=30` | Allow maven to download 30 artifacts at once. The default value is 5. This could speed up the build process by a lot when the maven cache was not previously populated. |
+| `-Pnative` | Build native module(s). So far it only affects `hdds-rocks-native` |
+
+### Build Output
+
+You can test the result of the compilation process by running a simple Ozone command which will display the Ozone version
+
+```bash
+./hadoop-ozone/dist/target/ozone-<version>-SNAPSHOT/bin/ozone version
+```
+
+The build process creates several important artifacts:
+
+- **Distribution Tarball**: `hadoop-ozone/dist/target/ozone-<version>.tar.gz` (when using `-Pdist`)
+- **Distribution Directory**: `hadoop-ozone/dist/target/ozone-<version>/`
+- **Individual Module JARs**: `hadoop-ozone/dist/target/ozone-<version>-SNAPSHOT/share/ozone/lib`
```

Comment:
```suggestion
The build process creates several important artifacts:

- **Distribution Tarball**: `hadoop-ozone/dist/target/ozone-<version>.tar.gz` (when using `-Pdist`)
- **Distribution Directory**: `hadoop-ozone/dist/target/ozone-<version>/`
- **Individual Module JARs**: `hadoop-ozone/dist/target/ozone-<version>/share/ozone/lib`

You can test the result of the compilation process by running a simple Ozone command which will display the Ozone version:

```bash
hadoop-ozone/dist/target/ozone-<version>/bin/ozone version
```

I think this order of presentation makes more sense.
- I removed `SNAPSHOT`, we can leave that as part of `<version>`. Not every build will be a snapshot build. 
- `./` is not technically required here since we are already providing a relative path to the executable.